app/components/market/submit/parameter-input.html

```diff
@@ -0,0 +1,35 @@
+<div [ngSwitch]="parameter.type">
+    <div *ngSwitchCase="NcjParameterExtendedType.string">
+        <md-input-container class="form-element">
+            <input mdInput type="text" name="parameter.name" [formControl]="parameterValue" [placeholder]="parameter.name">
```

Comment:
Is the 'placeholder' value shown in the UI? If so, should it be a friendly name such as "Parameter name"?